src/main/java/org/spongepowered/api/resource/ResourcePath.java

```diff
@@ -92,6 +93,54 @@ static ResourcePath parse(String path) throws IllegalArgumentException {
      */
     String getPath();
 
+    /**
+     * Gets the root path from this path. This is the same object as returned
+     * by the following code.
+     * <pre>
+     * List<ResourcePath> parents = getParents();
+     * if (parents.isEmpty()) {
+     *     return this;
+     * }
+     * return parents.get(parents.size() - 1);
+     * </pre>
+     *
+     * @return The root path
+     */
+    ResourcePath getRoot();
```

Comment:
Root would return `namespace:`, path is empty, if the docs are true this would return the parent path.

Reply:
This method does not appear meaningful to me. Why?